phat/tests/legoActions.test.ts

```diff
@@ -138,8 +143,18 @@ describe("Run lego actions", () => {
     this.timeout(500_000_000);
 
     before(async function () {
+      tagbag = await tagbagFactory.instantiate("default", [], {});
       // Deploy contracts
       lego = await legoFactory.instantiate("default", [], {});
+      await TxHandler.handle(
+        system.tx["system::setDriver"](
+          { gasLimit: "10000000000000" },
+          "TagStack",
+          tagbag.address.toHex()
+        ),
```

Comment:
driver TagStack is required for the logging span.

Reply:
Maybe it better move to the main repository.

Reply:
This PR may not be the optimal solution for logging; let's experiment with it first.